src/libs/actions/Policy.js

```diff
@@ -269,7 +269,11 @@ function createPolicyExpenseChats(policyID, members, betas) {
         );
         const optimisticCreatedAction = ReportUtils.buildOptimisticCreatedReportAction(optimisticReport.ownerEmail);
 
-        workspaceMembersChats.optimisticReportIDs[login] = optimisticReport.reportID;
+        workspaceMembersChats.reportIDs[login] = {
```

Comment:
I think you missed a spot when renaming here. Getting an error while testing this 
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/4073354/232907496-0f46a7bb-a6bb-464e-85d2-34864e26af03.png">
which is linking to this line 
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/4073354/232907543-f6279a08-b3fe-4dd2-b58d-667d5151c56e.png">

```suggestion
        workspaceMembersChats.reportCreationData[login] = {
```

Reply:
yep confirmed this change gets rid of the errors

Reply:
Sorry for that, commiting changes 🚀